### PR TITLE
Update ietf-te-path-computation.yang

### PR DESCRIPTION
**Pull request by efralaz**
Added returned-metrics-info grouping to be used both in input and output of stateless path computation rpc.

### CODE REVIEW

**Comment by italobusi** [comment]
I understand this pull request proposes a resolution for open issue #32

**Comment by efralaz** [comment]
Yes, it proposes a solution to issue #32, indeed.

File: ietf-te-path-computation.yang [hunk 1]
```diff
@@ -236,7 +249,7 @@ module ietf-te-path-computation {
       }
       uses te-types:generic-path-constraints;
       uses te-types:generic-path-optimization;
-      
+      uses returned-metrics-info;
```
Comment (italobusi) [inline ietf-te-path-computation.yang]: @efralaz : please check if there is a way to preclude the metric-value being specified in the request

Comment (efralaz) [reply ietf-te-path-computation.yang]: Not an economic way. It's just possible to create two different groupings, one for the request (without the returned values) and one for the reply (with the returned values, mandatory). 
As in many other places of the proposed standard (including te-tunnel and similia) optional fields are widely used for the purpose of defining one single grouping with multiple usage, I suggest to keep the same approach and specify in the description that the returned value is optional and must be not present (and if present it shall be ignored) in the request and must be present in the reply.

**Comment by italobusi** [comment]
Merged in pull request #41

**Pull request closed without merging**